.travis.yml

```diff
@@ -11,3 +11,7 @@ rvm:
   - 1.9.3
   - jruby-19mode
   - rbx-2.2.10
+
+matrix:
+  allow_failures:
+    - rvm: rbx-2.2.10
```

Comment:
Change this one to `rbx-2`, that means "the latest rbx 2", otherwise we constantly chase our tails.